src/lib/grammar.rs

```diff
@@ -386,4 +389,29 @@ mod test {
         assert_eq!(grm.prod_precs[6].unwrap(), Precedence{level: 4, kind: AssocKind::Nonassoc});
         assert!(grm.prod_precs[7].is_none());
     }
+
+    #[test]
+    fn test_prec_override() {
+        let grm = Grammar::new(&parse_yacc(&"
+            %start expr
+            %left '+' '-'
+            %left '*' '/'
+            %%
+            expr : expr '+' expr
+                 | expr '-' expr
+                 | expr '*' expr
+                 | expr '/' expr
+                 | %prec '-'  expr '*'
```

Comment:
Am I correct in that you want to make the precedence of unary minus the same as that of multiplication? Should't it then be `'-'  expr      %prec  '*'` (according to [this yacc guide](http://dinosaur.compilertools.net/yacc/))?